web/src/stream_settings_ui.js

```diff
@@ -1068,6 +1068,14 @@ export function unsubscribe_from_private_stream(sub) {
         ),
         html_body,
         on_click: unsubscribe_from_stream,
+        on_hide() {
+            // This is to avoid multiple tooltips with different messages. One
+            // on .sub_unsub_button and other on .sub_unsub_button_wrapper in
+            // case one uses sub_unsub_button on the right pane to unsubscribe
+            // from a private stream.
+            const $sub_unsub_button = settings_button_for_sub(sub);
+            $sub_unsub_button.trigger("blur");
+        },
```

Comment:
For private streams if the user is not subscribed to the stream the toggle subscription button on the right pane is disabled ans supports no pointer events. So to show tooltip with information `"Cannot subscribe to private stream {stream-name}"` we bind that tooltip to `.sub_unsub_button_wrapper`.

There was a corner case where if a user was first subscribed to a private stream uses toggle subscription button on right pane to unsubscribe then after the confirm dialog widget closes and user is unsubscribed user could see two tooltips if user hovers over the toggle subscription button, one that was already existing due to focus on the button when it was clicked for unsubscribing and second tooltip due to hover on sub_unsub_button.

To remove the stale tooltip we remove the focus from the button.

Reply:
I am not sure but can this behavior due to focus on the button be fixed by following approach similar to one used in #25381.